python/plugins/processing/gui/ProcessingToolbox.py

```diff
@@ -253,7 +253,7 @@ def executeAlgorithm(self):
                 feedback = MessageBarProgress(algname=alg.displayName())
                 ok, results = execute_in_place(alg, parameters, feedback=feedback)
                 if ok:
-                    iface.messageBar().pushSuccess('', self.tr('{} complete').format(alg.displayName()))
+                    iface.messageBar().pushSuccess('', self.tr('{algname} completed. {featurecount} feature(s) processed.').format(algname=alg.displayName(), featurecount=results['count']))
```

Comment:
What about using a %n tr count here? i.e. self.tr('{algname} completed. {featurecount}').format(algname=...,featurecount=self.tr('%n feature(s) processed','',results['count'])) ?

Reply:
Good point, I was trying to use `tr().arg()` which doesn't work ... But that's not actually required!